modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,606 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: REFERENCE
+[id="resolver-cluster-tasks-ref_{context}"]
+= Tasks provided in the {pipelines-shortname} namespace
+
+A {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortnames} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the Cluster resolver to access the tasks.
+
+Cluster task functionality is deprecated since version 1.10 and will be removed in a future release. If your pipelines use cluster tasks, you can re-create them using the tasks that are available with the cluster resolvers. However, certain changes are made in these tasks compared to older cluster tasks.
+
+You cannot specify a custom execution image in any of the tasks available in the {pipelines-shortnames} installation namespace. If you want to use a custom execution image, create a copy of the task and replace the image by editing the copy.
```

Comment:
Can we specify all those custom execution image which we removed here :thinking: 

I mean

You cannot specify a custom execution images (BUILDER_IMAGE, gitInitImage, KN_IMAGE, .......) in any of the tasks available in the {pipelines-shortnames} installation namespace. If you want to use a custom execution image, create a copy of the task and replace the image by editing the copy.